client/my-sites/upgrades/components/domain-warnings/style.scss

```diff
@@ -1,4 +1,4 @@
-.domain-warnings__unverified-domains {
+.domain-warnings-notice {
```

Comment:
I don't want to be a PITA, but since you're already changing the name of this CSS class, might as well change it to something that doesn't generate an eslint warning. According to our [CSS guidelines](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md), this should be `domain-warnings__notice`.